SPECS/k3s/k3s-1.25.5.spec

```diff
@@ -26,7 +26,7 @@ Source0:        https://github.com/k3s-io/%{name}/archive/refs/tags/v%{version}+
 # 3. cd %%{name}-%%{version}-k3s2
 # 4. go mod vendor
 # 5. pushd vendor
-# 6. git clone --single-branch --branch="v1.5.18" --depth=1 https://github.com/k3s-io/containerd
```

Comment:
Lets bump the release tag and add changelog as well.

Reply:
Done